.github/workflows/documentation.yml

```
@@ -14,21 +16,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4
         with:
-          python-version: "3.10"
-      - run: echo "cache_id=$(date --utc '+%V')" >> $GITHUB_ENV 
-      - uses: actions/cache@v3
+          enable-cache: true
+
+      - uses: actions/setup-python@v5
         with:
-          key: mkdocs-material-${{ env.cache_id }}
-          path: .cache
-          restore-keys: |
-            mkdocs-material-
+          python-version: '3.10'
       
       - name: Dependencies
-        run: |
-          pip install -r docs/requirements.txt
+        run: make install
```

I don't see this set up anywhere? Alternatively:

```suggestion
        run: uv sync
```